inc/wp-custom-menu.php

```diff
@@ -75,7 +75,6 @@ function uds_wp_get_menu_formatted_array( $menu_name ) {
 					$menu[ $top_menu ]['children'][ $m->menu_item_parent ]['children'][ $m->ID ] = $column[ $m->ID ];
```

Comment:
Got some errors when tries to move some items one level down "as grandchild": 
Undefined index: parent in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 74
Undefined index: title in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 114 
Undefined index: url in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 122 
Undefined index: url in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 139 
Undefined index: cta_button in /app/wp-content/themes/UDS-WordPress-Theme/asu-navigation-menu.php on line 49 
Undefined index: cta_color in /app/wp-content/themes/UDS-WordPress-Theme/asu-navigation-menu.php on line 50
Undefined index: url in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 223 
Undefined index: title in /app/wp-content/themes/UDS-WordPress-Theme/inc/wp-custom-menu.php on line 223

Reply:
It looks like Zainab discovered a bug with our menu code here that's outside the scope of this PR for highlighting top level items (in fact, the top-level highlighting still works even though the errors appear in the menu!). I'm going to create a new issue in the project for handling this and go ahead and merge this PR so that we tackle this issue on an updated codebase.